consensus/spos/bls/subroundSignature.go

```diff
@@ -258,13 +258,21 @@ func (sr *subroundSignature) doSignatureConsensusCheck() bool {
 		return false
 	}
 
+	isSelfInConsensusGroup := sr.IsNodeInConsensusGroup(sr.SelfPubKey()) || sr.IsMultiKeyInConsensusGroup()
+	if !isSelfInConsensusGroup {
+		log.Debug("step 2: subround has been finished",
```

Comment:
if we have this early check here, i think that the reference check to `!isSelfInConsensusGroup` on L302 is not really needed anymore